### PR TITLE
BI-10492 Redirect to stop page with controller

### PR DESCRIPTION
**Pull request by mattch1**
Need to separate out the controller and stop page urls, but they still use the same template

### CODE REVIEW

File: src/routes/routes.ts [hunk 1]
```diff
@@ -28,7 +28,8 @@ import * as wrongStatementOfCapitalRoute from "../controllers/incorrect-informat
 import * as wrongShareholdersRoute from "../controllers/incorrect-information/wrong.shareholders.controller";
 import * as wrongRegisteredOfficeAddressRoute from "../controllers/incorrect-information/wrong.registered.office.address.controller";
 import * as wrongRegisterLocationsRoute from "../controllers/incorrect-information/wrong.registers.controller";
-import * as wrongDetailsRoute from "../controllers/incorrect-information/wrong.details.controller";
+import * as wrongDetailsRoute from "../controllers/incorrect-information/wrong.officer.details.controller";
```
Comment (mwestacott) [inline src/routes/routes.ts:31]: Could you import this with the name 'wrongOfficerDetailsRoute' which makes clear what it is? Consistent then with 'wrongPscDetailsRoute'.

File: src/types/page.urls.ts [hunk 2]
```diff
@@ -79,5 +79,7 @@ export const WRONG_RO =  ACTIVE_SUBMISSION_BASE + Templates.WRONG_RO;
 export const WRONG_RO_PATH = CONFIRMATION_STATEMENT + WRONG_RO;
 export const WRONG_REGISTER_LOCATIONS = ACTIVE_SUBMISSION_BASE + Templates.WRONG_REGISTER_LOCATIONS;
 export const WRONG_REGISTER_LOCATIONS_PATH = CONFIRMATION_STATEMENT + WRONG_REGISTER_LOCATIONS;
-export const WRONG_DETAILS = ACTIVE_SUBMISSION_BASE + Templates.WRONG_DETAILS;
+export const WRONG_DETAILS = ACTIVE_SUBMISSION_BASE + "wrong-officer-details";
```
Comment (mwestacott) [inline src/types/page.urls.ts:82]: So would then suggest refactoring/renaming this to WRONG_OFFICER_DETAILS too.

Comment (mattch1) [reply src/types/page.urls.ts:82]: The template is used by both the officers and psc page so needs to stay as generic as possible.

Comment (mattch1) [reply src/types/page.urls.ts]: Ignore my reply changing this.

**Review by mwestacott — APPROVED** [review]

**Pull request merged**